src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -0,0 +1,55 @@
+
+<div class="challenge-card">
+  <div class="bottom-card-container ev-card-panel">
+    <div class="row">
+      <div class="col-md-8 col-sm-8">
+        <span class="fs-18">
+          <span class="fw-semibold">Challenge Settings</span>
+        </span>
+      </div>
+    </div>
+    <br>
+    <div class="row settings-section">
+      <div class="banned-email-title col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-regular">Banned emails Ids:</span>
+      </div>
+      <div class="banned-email-input col-lg-7 col-md-7 col-sm-9 col-xs-9">
+        <div class="">
+          <span *ngIf="isBannedEmailInputVisible" class="display-inline">
+            <span class="col-md-7 col-sm-7 col-xs-7 li-mr">
+              <app-input [label]="''" [value]="bannedEmailIds"
+                [name]="'bannedEmailIds'"
+                (input)="validateInput($event.target.value)"
+                [type]="'textarea'" [icon]="''" #formgroup>
+              </app-input>
+            </span>
+            <span class="col-md-5 col-sm-5 col-xs-5 li-mr display-inline">
+              <a class="pointer li-mr" (click)="updateBannedEmailList()">
+                <i class="fa fa-check green-text fa-lg" aria-hidden="true"></i>
+              </a>
+              <a class="pointer li-mr" (click)="isBannedEmailInputVisible=
+                false">
+                <i class="fa fa-times red-text fa-lg" aria-hidden="true"></i>
+              </a>
+            </span>
+          </span>
+          <span *ngIf="!isBannedEmailInputVisible" class="display-inline">
+            <span class="row">
+              <span *ngIf="challenge.banned_email_ids" class="col-md-7 col-sm-7 li-mr">
+                <span style="white-space: pre-line;" *ngFor="let participant_email of challenge.banned_email_ids"
+                  class="settings-value">{{participant_email}}<wbr></span>
+              </span>
+              <span class="col-md-3 col-sm-3">
+                <a class="pointer li-mr" (click)="isBannedEmailInputVisible=
+                  true">
+                  <i class="fa fa-pencil" aria-hidden="true">
+                  </i>
+                </a>
+              </span>
+            </span>
+          </span>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
Add a new-line at the end of all newly added files.
```suggestion
</div>

```

Reply:
@galipremsagar done